src/osg_display/display_graph.py

```diff
@@ -185,7 +181,10 @@ def run(self, sect, mode="hourly"):
             name, tmpname = get_files(self.cp, sect, format=format)
             self.build_canvas(format=format)
             self.draw()
-            fd = open(tmpname, 'w')
+            if format in ['SVG']: # String output formats
+                fd = open(tmpname, 'w')
+            else: # Byte output formats
+                fd = open(tmpname, 'wb')
```

Comment:
Just curious - is the output broken for svg with `wb` and/or non-svg without `wb`?

Reply:
Yes indeed. Python 3 threw an error if the file handle didn't match the data type.